pypeit/scripts/view_fits.py

```diff
@@ -126,3 +129,6 @@ def main(args):
         display.show_image(img, chname=args.chname)
 
 
+        if args.embed:
+            embed()
```

Comment:
can you add  `header='line 133 of view_fits.py`
  so people know where they landed?

Reply:
I would instead use `pypeit.utils.embed_header()`.  It automatically pulls this info from the stack so it's always correct.

Reply:
Done and made the same changed to pypeit_show_2dspec which also has an embed option.